dace/codegen/targets/cuda.py

```diff
@@ -2096,7 +2096,7 @@ def get_next_scope_entries(self, dfg, scope_entry):
 
         # Get all non-sequential scopes from the same level
         all_scopes = [
```

Comment:
I am not exactly sure why this does not use `scope_subgraph` instead.

Reply:
hmm, I'll try to look into this later. But for now I'd like to wrap up this PR. If we should change this, I think it should be in a separate PR.

But thanks for pointing it out. Looks like there's a potential to improve the code there.